docs/ref/validators.txt

```diff
@@ -145,6 +145,23 @@ to, or in lieu of custom ``field.clean()`` methods.
         ``['localhost']``. Other domains that don't contain a dot won't pass
         validation, so you'd need to whitelist them as necessary.
 
+    The :class:`EmailValidator` works in accordance with the `HTML5 Email
+    validation standard`_.
+
+    .. _HTML5 Email validation standard: https://html.spec.whatwg.org/multipage/forms.html#valid-e-mail-address
+
+    .. versionchanged:: 2.0
+
+        In older version, Email validation was done with Django's own
+        logic. It uses the HTML5 Email validation standard from 2.0
+	onwards.
```

Comment:
Okay. Does `Since 2.0, Django uses the standard HTML5 Email validation logic.` sound better than
```
In older version, Email validation was done with Django's own logic. It uses the HTML5 Email validation standard from 2.0 onwards.
```
?